core/storage/base_model/gae_models.py

```diff
@@ -895,26 +893,24 @@ def get_version(cls, entity_id, version_number, strict=True):
         Raises:
             Exception: This model instance has been deleted.
         """
-        # pylint: disable=protected-access
         current_version_model = cls.get(entity_id, strict=strict)
 
         if current_version_model is None:
             return None
 
-        current_version_model._require_not_marked_deleted()
+        current_version_model._require_not_marked_deleted()  # pylint: disable=protected-access
 
         snapshot_id = cls.get_snapshot_id(entity_id, version_number)
 
         try:
-            return cls(
+            return cls(  # pylint: disable=protected-access
```

Comment:
Can this go on the right of row 909 instead? If not, it's OK.

Reply:
I tried moving it to line 909 but that didn't work. So I left it as it is.

Reply:
OK, thanks for checking. Sounds good.